docker/grafana/dashboards/ontime_database_carrier_and_airport_stats.json

```diff
@@ -0,0 +1,1349 @@
+{
+  "__inputs": [
+    {
+      "name": "DS_GITHUB-DEMO-CLUSTER-ALTINITY",
```

Comment:
You propose dashboard which saved as `Export for sharing externally`.
We use `docker/grafana/dashboards` to deploy dashboards with direct names of data sources with reference already present in `docker/grafana/dasources/`

Let move it to `/public_dashboards/` instead of `docker/grafana/dashboards`

Reply:
created /public_dashboards directory, moved JSON file there